enterprise/ha/src/main/java/org/neo4j/kernel/ha/HaSettings.java

```diff
@@ -101,7 +101,15 @@
         @Description("Round robin")
         round_robin,
 
-        @Description("Fixed")
-        fixed
```

Comment:
Ugh, I only thought of the setting-via-config-file approach, where the migrator will make the old setting work. Didn't think of this java API also being public. I'll put it back in and deprecate it.